README.md

```diff
@@ -14,5 +14,16 @@ The features of the report includes:
 * [**User Guide**](docs/UserGuide.md)
 * [**Developer Guide**](docs/DeveloperGuide.md)
 
+## About Us
+RepoSense is a project based in the [National University of Singapore, School of Computing](http://www.comp.nus.edu.sg/), and is funded by a grant from [NUS Center for Development of Teaching and Learning](http://www.cdtl.nus.edu.sg/).
+
+* [**Aditya Agarwal**](https://github.com/yong24s): _Project Member_ since May 2018
```

Comment:
~~yong24s~~ adityaa1998

Reply:
fixed, sorry must have accidentally pasted the wrong link.